content/en/docs/examples/microservices-istio/enable-istio-all-microservices/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Enable Istio on all the microservices
+overview: Enable Istio on your whole application.
+weight: 70
+
+---
+
+Previously, you enabled Istio on a single microservice, `productpage`. You can
+proceed to enable Istio on the microservices incrementally to get the Istio
+functionality for more microservices. For the purpose of this tutorial, you will
+enable Istio on all the remaining microservices in one step.
+
+1.  For the purpose of this tutorial, scale the deployments of the microservices
+    down to 1:
+
+    {{< text bash >}}
+    $ kubectl scale deployments --all --replicas 1
+    {{< /text >}}
+
+1.  Redeploy the Bookinfo application, Istio-enabled. `productpage` will not be
+    redeployed since it already has Istio injected, and its pods will not be
+    changed. This time you will use only a single replica of a microservice,
+    scaling of the microservices is orthogonal to the demonstrated Istio
+    features.
+
+    {{< text bash >}}
+    $ curl -s {{< github_file >}}/samples/bookinfo/platform/kube/bookinfo.yaml | istioctl kube-inject -f - | kubectl apply -l app!=reviews -f -
+    $ curl -s {{< github_file >}}/samples/bookinfo/platform/kube/bookinfo.yaml | istioctl kube-inject -f - | kubectl apply -l app=reviews,version=v2 -f -
+    service "details" unchanged
+    deployment "details-v1" configured
+    service "ratings" unchanged
+    deployment "ratings-v1" configured
+    service "productpage" unchanged
+    deployment "productpage-v1" configured
+    deployment "reviews-v2" configured
+    {{< /text >}}
+
+1.  Access the application's webpage several times. Note that Istio was added
+    **transparently**, the original application did not change. It was added on
+    the fly, without the need to undeploy and redeploy the whole application,
```

Comment:
this seems misleading, we do need to restart the application

Reply:
fixed